distro/configuration/concepts/hiv_and_art_follow_up_concepts.csv

```diff
@@ -132,55 +134,55 @@ da3c6710-c431-4582-a444-a466d54693ec,,5c = ABC/3TC (120/60) + RAL,5c = ABC/3TC (
 50b60d77-186d-4a0d-8784-659ee2d60ec9,,5i = ABC/3TC + LPV/r,5i = ABC/3TC + LPV/r,,,,,Misc,N/A,
 78e49624-0e33-4374-93b7-60b132b26dae,,5j = AZT/3TC (120/60)+DTG10,5j = AZT/3TC (120/60)+DTG11,,,,,Misc,N/A,
 d75c973c-06d4-43d2-a57d-6340fc8c0100,,Other ART Regimen,Other ART Regimen,,,,,Misc,N/A,
-23322fd6-3dbb-410e-8bee-6210dfcd5f71,,ARV Regimen,ARV Regimen,ARV Regimen,,,,Question,Coded,"1a = AZT/3TC + EFV;
```

Comment:
Was the update to this code snippet necessary @PiusKariuki? If not, I suggest you revert it. Some key reports rely on these values staying as they are. Any inadvertent change may break the reports. So, there is no need to tough them unless absolutely necessary, in which case the corresponding updates have to be ,ade on the reports